assets/_scss/_navigation.scss

```
@@ -278,5 +320,14 @@ Overlay : allways ***********************
  *  Vertical Navigation common styles
  */
  :where( .wp-block-navigation.is-vertical ) {
-	// nothing now
+
+	// エディタで open-on-click の時にコアが出力してくる padding:0 の上書き
```

Stylelint (17.11.1)

[error] Unexpected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)